tss-esapi/src/context/general_esys_tr.rs

```diff
@@ -388,6 +393,44 @@ impl Context {
         TpmHandle::try_from(tpm_handle)
     }
 
-    // Missing function: Esys_TR_Serialize
-    // Missing function: Esys_TR_Deserialize
+    /// Store the `ObjectHandle` in a buffer
```

Comment:
Could you please make these a bit more descriptive, as the name of the function would be somewhat confusing for a library user (since we don't expose the idea of a "TPM Resource" under that name). Perhaps something like:

```
Serialize the metadata of the object identified by `handle` into a new buffer.

This can subsequently be used to recreate the object in a new context in the future.
```

Also, similar request for the deserialize method

Reply:
@ionut-arm done